### PR TITLE
Added More Robust Example

### PR DESCRIPTION
**Pull request by askpatrickw**
A little more help for those getting started. This would have saved me some time this week.

### CODE REVIEW

File: docs/widget.rst [hunk 1]
```diff
@@ -17,10 +17,24 @@ In Adminsite
...
     from .models import WeatherStation
+    from .models import WeatherSensor
 
-
+    # A Basic Example
     admin.site.register(WeatherStation, LeafletGeoAdmin)
 
+   """A More Complicated Example
+      Notice ModelAdmin(admin.ModelAdmin) also has to change to modelAdmin(LeafletGeoAdmin)
+      @admin.register() decorator not required for Django-Leaflet, but is for inlines.
```
Comment (Gagaro) [inline docs/widget.rst]: Don't you mean the other way around? register is required for the `TabularInline` but is for the `LeafletGeoAdmin`?

File: docs/widget.rst [hunk 2]
```diff
@@ -8,7 +8,7 @@ It embeds *Leaflet.draw* in version *0.4.0*.
 .. image :: https://f.cloud.github.com/assets/546692/1048836/78b6ad94-1094-11e3-86d8-c3e88626a31d.png
 
 
-In Adminsite
+In admin.py
 ------------
```
Comment (Gagaro) [inline docs/widget.rst]: Please match the number of `-` under the title.

File: docs/widget.rst [hunk 3]
```diff
@@ -33,7 +47,7 @@ A mixin is also available for inline forms:
         model = PoiLocation
 
 
-In forms
+In forms.py
 --------
```
Comment (Gagaro) [inline docs/widget.rst]: Please match the number of `-` under the title.

File: docs/widget.rst (hunk 1 above)
Comment (W7PEA) [reply docs/widget.rst]: No. I tested this. The decorator is required for the inline only.

File: docs/widget.rst (hunk 2 above)
Comment (W7PEA) [reply docs/widget.rst]: Sorry, I don't understand and don't know RST. Are you saying the "-" line should be the same length and the line above it?

Comment (Gagaro) [reply docs/widget.rst]: Yes. cf http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#sections

File: docs/widget.rst (hunk 1 above)
Comment (Gagaro) [reply docs/widget.rst]: Then there is something I don't understand with what you wrote. The `LeafletGeoAdmin` is correctly decorated but does it have to do with the inlines?

Comment (W7PEA) [reply docs/widget.rst]: The Decorator is not required for django-leaflet, but _if_ you have an inline it is required.

Comment (Gagaro) [reply docs/widget.rst]: It should be required with or without inline (or `admin.site.register`).

File: docs/widget.rst (hunk 2 above)
Comment (mfrasca) [reply docs/widget.rst]: it doesn't really matter, as long as it's equal or more.  it works this way too.

**Comment by mfrasca** [comment]
I think it's a pity to have users contributions get outdated on such smallish base, like the number of dashes below a title.

**Comment by askpatrickw** [comment]
yeah.. I gave up.

**Pull request closed without merging**